common/main/java/com/couchbase/lite/internal/core/C4Database.java

```diff
@@ -51,13 +51,11 @@
     "PMD.ExcessiveParameterList",
     "PMD.CyclomaticComplexity"})
 public abstract class C4Database extends C4NativePeer {
-    public static final boolean VERSION_VECTORS_ENABLED = false;
-
     @VisibleForTesting
-    static final int DB_FLAGS = C4Constants.DatabaseFlags.CREATE;
+    public static final String DB_EXTENSION = ".cblite2";
 
     @VisibleForTesting
-    public static final String DB_EXTENSION = ".cblite2";
+    static final int DB_FLAGS = C4Constants.DatabaseFlags.CREATE;
```

Comment:
Why are these flags not including versionvector by default?

Reply:
Because the VersionVector flag is added in the code you highlighted below.